source/documentation/services/github/jml4-tool.html.md.erb

```diff
@@ -0,0 +1,53 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Monthly Upload of Github User Data for JML4 Tool
+last_reviewed_on: 2024-01-05
+review_in: 3 months
+---
+
+# Monthly Upload of Github User Data for JML4 Tool
+
+## Overview
+
+This runbook relates to a trial of the [JML4 Tool](https://app.powerbi.com/groups/me/reports/ad061100-2417-4616-9481-efb533fb88d9/ReportSectionbe145fa8de8a4c99e949?experience=power-bi) (JML = Joiners, Movers, Leavers). The JML4 Tool is a user account remediation tool that collates user data from multiple sources to identify leavers.
+
+So that we can use this tool we provide the Data Controller with a monthly extract of Github user data.
```

Comment:
Think we need a comma, not entirely sure about the right interpretation 🙈 

It's a requirement to provide the data is my initial interpretation 👇 

`So that we can use this tool, we provide the Data Controller with a monthly extract of Github user data.`

Reply:
I've changed this sentence. Review when I've fix the links.